app/templates/base.html

```diff
@@ -31,20 +31,20 @@
                         </a>
                     </li>
                     <li class="govuk-header__navigation-item">
-                        <a class="govuk-header__link" href="#2">
+                        <a class="govuk-header__link" href="/">
                             Upcoming
                         </a>
                     </li>
                     <li class="govuk-header__navigation-item">
-                        <a class="govuk-header__link" href="#3">
+                        <a class="govuk-header__link" href="/my-masterclasses">
```

Comment:
Below you use `url_for`, which I think is a better pattern. Could you update these for consistency?